xgb.go

```diff
@@ -366,9 +398,8 @@ func (c *Conn) writeBuffer(buf []byte) error {
 	if _, err := c.conn.Write(buf); err != nil {
 		Logger.Printf("A write error is unrecoverable: %s", err)
```

Comment:
I wonder if `broadcastDone` should be called here, too. `writeBuffer` returns an `error` which is ignored. Sending another request will likely fail.